### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by yehche01**
# Checklist:
- [x] My code follows the style guidelines of this project (IntelliJ Default)
- [x] My changes generate no new warnings
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/main/java/tw/b2e/receipt/ReceiptRouter.java [hunk 1]
```diff
@@ -6,4 +6,11 @@
 
 @Component
 public class ReceiptRouter implements Router<SlashCommandRequest> {
+
```
Comment (Jian-Min-Huang) [inline src/main/java/tw/b2e/receipt/ReceiptRouter.java]: Add @Override

File: src/main/java/tw/b2e/receipt/PrizeNumberObject.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    public PrizeNumberObject(String year, String month){
+        this.year = year;
+        this.month = month;
+        this.url = url + year + month;
+        jsoupRes = this.getPrizeNumber();
+    }
+
+    public boolean getPrizeNumber(){
+        try {
+
```
Comment (Jian-Min-Huang) [inline src/main/java/tw/b2e/receipt/PrizeNumberObject.java]:
```suggestion
```

File: src/main/java/tw/b2e/receipt/ReceiptRouter.java (hunk 1 above)
Comment (Jian-Min-Huang) [inline src/main/java/tw/b2e/receipt/ReceiptRouter.java]:
```suggestion
@Override
```

File: pom.xml [hunk 3]
```diff
@@ -42,6 +42,14 @@
                 </exclusion>
             </exclusions>
         </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.jsoup/jsoup -->
```
Comment (Jian-Min-Huang) [inline pom.xml]:
```suggestion
```

File: pom.xml [hunk 4]
```diff
@@ -42,6 +42,14 @@
...
+        <dependency>
+            <groupId>org.jsoup</groupId>
+            <artifactId>jsoup</artifactId>
+            <version>1.13.1</version>
+        </dependency>
+
```
Comment (Jian-Min-Huang) [inline pom.xml]:
```suggestion
```

File: src/main/java/tw/b2e/receipt/PrizeNumberObject.java [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+                String[] tempG9 = group9.text().trim().split("、");
+                for(String s : tempG9){
+                    if(!"".equals(s)){
+                        addSixPrizeNum.add(s);
+                    }
+                }
+            }
+
+            return true;
+
```
Comment (Jian-Min-Huang) [inline src/main/java/tw/b2e/receipt/PrizeNumberObject.java]:
```suggestion
```

**Review by Jian-Min-Huang — COMMENTED** [review]
@yehche01 pls check or you can apply directly 😎

**Review by Jian-Min-Huang — APPROVED** [review]
LGTM

**Review by jimlee0507 — APPROVED** [review]
LGTM

**Pull request merged**